public_html/layouts/basic/modules/Settings/PDF/resources/Import.js

```diff
@@ -0,0 +1,17 @@
+/* {[The file is published on the basis of YetiForce Public License 3.0 that can be found in the following directory: licenses/LicenseEN.txt or yetiforce.com]} */
+'use strict';
+
+jQuery.Class('Settings_PDF_Import_Js', {}, {
+  /**
+   * Function to register event for PDF import
+   */
+  registerEvents: function() {
+    let form = $('.js-validation-engine');
+    form.validationEngine();
+  }
+});
+
+jQuery(document).ready(function() {
```

Comment:
We don't need to call registerEvents() class function manually (it appears only in old code to refactor), app.getPageController do this for us ;)

Reply:
hurg! I didn't know this: great!
I'm going to remove ;)

Reply:
done